util/tasks.go

```diff
@@ -96,6 +98,35 @@ func Reset() {
 	}
 }
 
+func Web(url string){
```

Comment:
I would put everything web related into a separate file called "web.go" (still under the same util package). Makes it easier to organize when more web-related features and options are added.

Reply:
Done!